dotnet-client-libraries/MoreLINQ/MoreLINQ.Tests/FlightTests.cs

```diff
@@ -0,0 +1,344 @@
+﻿using Bogus;
+using ExpandingLIQNwithMoreLINQ;
+
+namespace ExpandingLINQwithMoreLINQ.Tests
+{
+    public class FlightTests
+    {
+        [Fact]
+        public void GivenValidParameters_WhenConstructorIsInvoked_ThenValidFlightIsReturned()
+        {
+            // Arrange
+            int flightNumber = 1;
+            string departureCity = "Sofia, Bulgaria";
+            string arrivalCity = "Nis, Serbia";
+            DateTime departureTime = DateTime.UtcNow;
+            DateTime arrivalTime = DateTime.UtcNow;
```

Comment:
Since you're doing this setup in each test, please create a method that can help you to not repeat the code each time.

Reply:
The code is updated now.